dialogue-java-client/src/main/java/com/palantir/dialogue/JavaChannels.java

```diff
@@ -75,4 +81,13 @@ private static SSLContext createSslContext(TrustManager trustManager) {
             throw new IllegalStateException(e);
         }
     }
+
+    // Same default thread pool with tracing enabled
+    private static ExecutorService createExecutor() {
```

Comment:
implementing this here, rather than higher up means we need to also implement it in other Channel impls (eg: OkHttp).